main.py

```diff
@@ -1,30 +1,50 @@
-import hug
+#import hug
+import sqlalchemy as db
 
 
-@hug.response_middleware()
-def process_data(request, response, resource):
-    response.set_header('Access-Control-Allow-Origin', '*')
-    response.set_header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS')
-    response.set_header('Access-Control-Allow-Headers', 'Content-Type')
+engine = db.create_engine('sqlite:///goals.sqlite')
```

Comment:
We should use a name that represents the project and not just one table.
For example
```suggestion
engine = db.create_engine('sqlite:///411-tracker-api.sqlite')
```